src/ZoneServer/Buffs/Handlers/Scout/Assassin/Hasisas.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Reflection.Metadata;
+using Melia.Shared.Game.Const;
+using Melia.Zone.Buffs.Base;
+
+namespace Melia.Zone.Buffs.Handlers.Scout.Assassin
+{
+	/// <summary>
+	/// Handle for the Bear Buff, which increases the target's defense.
+	/// </summary>
+	[BuffHandler(BuffId.Hasisas_Buff)]
+	public class Hasisas : BuffHandler
+	{
+		private const string aspdVarName = "Melia.AttackSpeedBonus";
```

Comment:
Our coding conventions dictate that consts, statics, and public fields should be formatted in `PascalCase`.